public/src/components/teacher/CourseGroup.tsx

```diff
@@ -229,6 +236,60 @@ export class CourseGroup extends React.Component<ICourseGroupProps, ICourseGroup
         });
     }
 
+    private generateGroupButtons(group: Group): ILink[] {
+        const links = [];
+        switch (group.getStatus()) {
+            case Group.GroupStatus.PENDING:
+                links.push({
+                    name: "Approve",
+                    extra: "primary",
+                    uri: "approve",
+                    description: "list-btn",
+                }, {
+                    name: "Edit",
+                    extra: "primary",
+                    uri: "edit",
+                }, {
+                    name: "Delete",
+                    extra: "danger",
+                    uri: "delete",
+                    description: "list-btn",
+                });
+                break;
+            case Group.GroupStatus.APPROVED:
+                this.state.editing ? links.push({
+                    name: "Edit",
+                    extra: "primary",
+                    uri: "edit",
+                    description: "list-btn",
+                }, {
+                    name: "Delete",
+                    extra: "danger",
+                    uri: "delete",
+                    description: "list-btn",
+                }) : links.push({
+                    name: "Approved",
+                    extra: "light",
```

Comment:
Shouldn't this have `uri` field?

Reply:
No, it's those buttons: 
![image](https://user-images.githubusercontent.com/26818902/75233330-f9840800-57b8-11ea-855c-76e2aeade818.png)

They don't have any uri links.